package.json

```diff
@@ -54,6 +54,7 @@
     "@aws-cdk/aws-apigatewayv2-integrations": "1.105.0",
     "@aws-cdk/aws-lambda": "1.105.0",
     "@aws-cdk/aws-lambda-nodejs": "1.105.0",
+    "@aws-cdk/aws-rds": "^1.107.0",
```

Comment:
Just curious, was this import required for this PR?

Reply:
Probably not no